plugin-gradle/src/main/java/com/diffplug/gradle/spotless/SpotlessTask.java

```diff
@@ -303,9 +304,21 @@ private void check(Formatter formatter, List<File> outOfDate) throws Exception {
 	/** Returns an exception which indicates problem files nicely. */
 	GradleException formatViolationsFor(Formatter formatter, List<File> problemFiles) {
 		return new GradleException(DiffMessageFormatter.builder()
-				.runToFix("Run 'gradlew spotlessApply' to fix these violations.")
+				.runToFix("Run '" + calculateGradleCommand() + " " + getTaskPathPrefix() + "spotlessApply' to fix these violations.")
 				.formatter(formatter)
 				.problemFiles(problemFiles)
 				.getMessage());
 	}
+
+	private String calculateGradleCommand() {
+		return Os.isFamily(Os.FAMILY_WINDOWS)
+				? "gradlew.bat"
```

Comment:
On windows, I've always seen and used `gradlew`, never `gradlew.bat`.

Reply:
hmm. I'd go with gradlew.bat as this is the correct wrapper script on windows

Reply:
Looks like you're right, `.bat` it is! https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:using_wrapper